src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/jpa/repository/ChildGroupRepository.kt

```diff
@@ -1,11 +1,15 @@
 package uk.gov.justice.digital.hmpps.externalusersapi.jpa.repository
```

Comment:
Do we need to set CoroutineCrudRepository approach or query based approach
Query based approach , can leave Model objects structured as OR models

Reply:
No Action - we are using CoroutineCrudRepository and only using query based approach (using databaseclient) when this is not possible (e.g. joining across multiple tables)